contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -2505,13 +2505,13 @@ SELECT
    CAST(0 as INT) AS unique_index_id,
```

Comment:
Let's add upgrade tests for this view as well if we have not done so already

Reply:
Done. Also added upgrade tests for 3 other views added to this PR who's column datatypes were changed in a similar manner.